src/utils/timeSeriesResultsByOption.js

```diff
@@ -4,7 +4,7 @@ import { FV, PMT } from 'formulajs/lib/financial'
 const COMPOUND_FREQUENCY = 12
```

Comment:
just replaced loanAmount with loanAmt to keep consistency in this file, no other changes.